Cargo.toml

```
@@ -11,6 +11,7 @@ include = ["src/**/*", "LICENSE-*", "*.md"]
 [dependencies]
 clap = { version = "~4.4", features = ["derive", "std", "help"], default-features = false }
 git2 = { version = "~0.18", features = ["ssh", "https"], default-features = false }
+auth-git2 = "0.5.0"
```

```suggestion
auth-git2 = "~0.5"
```

We're used to pin the dependencies that way within the project.